spec/index.bs

```diff
@@ -1445,6 +1451,11 @@ The capabilities of a [=device=] are exactly the ones which were requested in
 {{GPUAdapter/requestDevice()}}. These capabilities are enforced regardless of the
 capabilities of the [=adapter=].
 
+Additionally, WebGPU [=adapters=] support [=behaviors=], which describe WebGPU functionality
```

Comment:
Per [3/08 meeting](https://docs.google.com/document/d/12Wamx6G5wyuL-Lqs0hPZ0eintbUuwvWChe5rYWDVQWQ/edit#heading=h.axbubku1nyuw) we think this should be global constant on `navigator.gpu.whatever`, not a member of the adapter.